src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -21,7 +21,7 @@ import {
   BULK_EDIT_IDENTIFIERS,
   EDIT_CAPABILITIES,
   BULK_EDIT_QUERY,
-  CRITERIES, CAPABILITIES, translationSuffix,
+  CRITERION, CAPABILITIES, translationSuffix,
```

Comment:
Could you place each import on its own line for consistency?
```suggestion
  CRITERION,
  CAPABILITIES,
  translationSuffix,
```

Reply:
Thanks @mariia-aloshyna , already fixed